web/themes/custom/hatter/js/map.js

```diff
@@ -0,0 +1,18 @@
+
+(function ($) {
+
+  Drupal.behaviors.map = {
+    attach: function (context, settings) {
+      mapboxgl.accessToken = 'pk.eyJ1IjoibWlkY2FtcCIsImEiOiJjaXV2czRteHQwNGhkMm9zNWp4bWhndTk0In0.F5jwcKus8Rzx5r8Ga-CRQA';
+      var map = new mapboxgl.Map({
+        container: 'map', // container id
+        style: 'mapbox://styles/midcamp/ciuvsjxje00312ipqrfyxm4di', //stylesheet location
+        center: [-87.66, 41.924], // starting position
```

Comment:
At some point this will need to change to a geo-coded version of the venue address. I think we can postpone that until later, but wanted to flag.

Reply:
I wonder if we could use geocoder to just create the coordinates of the map based on the address and pull that information in here. No extra work for content generators and also working maps.